src/main/java/parser/StockValidator.java

```diff
@@ -163,9 +163,11 @@ public static boolean isValidMaxQuantity(Ui ui, String maxQuantityString) {
      */
     public static boolean isValidColumn(Ui ui, String columnName) {
         String[] columnAlias = new String[]{CommandParameters.STOCK_ID, CommandParameters.NAME, CommandParameters.PRICE,
-            CommandParameters.QUANTITY, CommandParameters.EXPIRY_DATE, CommandParameters.DESCRIPTION,
-            CommandParameters.MAX_QUANTITY};
-        if (Arrays.asList(Stock.COLUMNS).contains(columnName) || Arrays.asList(columnAlias).contains(columnName)) {
+                                            CommandParameters.QUANTITY, CommandParameters.EXPIRY_DATE,
```

Comment:
Is this indentation correct? I think indentation for wrapped lines should only be 8 spaces more than parent line.
https://se-education.org/guides/conventions/java/basic.html